Cargo.lock

```diff
@@ -9,54 +9,54 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "3720d0064a0ce5c0de7bd93bdb0a6caebab2a9b5668746145d7b3b0c5da02914"
 dependencies = [
  "actix-rt",
- "bitflags 1.2.1",
```

Comment:
This Cargo.lock has too many changes, I think you accidentally updated some other dependencies? You can revert it using

    git checkout upstream/master -- Cargo.lock

And compiling again.